.azure-pipelines/ci-build.yml

```diff
@@ -2,354 +2,344 @@
-      - pwsh: dotnet pack $(Build.SourcesDirectory)/src/Microsoft.OpenApi.Readers/Microsoft.OpenApi.Readers.csproj -o $(Build.ArtifactStagingDirectory) --configuration $(BuildConfiguration) --no-build --include-symbols --include-source /p:SymbolPackageFormat=snupkg
-        displayName: 'pack Readers'
-
-      # Pack
-      - pwsh: dotnet pack $(Build.SourcesDirectory)/src/Microsoft.OpenApi.Hidi/Microsoft.OpenApi.Hidi.csproj -o $(Build.ArtifactStagingDirectory) --configuration $(BuildConfiguration) --no-build --include-symbols --include-source /p:SymbolPackageFormat=snupkg
-        displayName: 'pack Hidi'
-
-      - task: EsrpCodeSigning@2
-        displayName: 'ESRP CodeSigning Nuget Packages'
-        inputs:
-          ConnectedServiceName: 'microsoftgraph ESRP CodeSign DLL and NuGet (AKV)'
-          FolderPath: '$(Build.ArtifactStagingDirectory)'
-          Pattern: '*.nupkg'
-          signConfigType: inlineSignParams
-          UseMinimatch: true
-          inlineOperation: |
-            [
-                {
-                    "keyCode": "CP-401405",
-                    "operationSetCode": "NuGetSign",
-                    "parameters": [ ],
-                    "toolName": "sign",
-                    "toolVersion": "1.0"
-                },
-                {
-                    "keyCode": "CP-401405",
-                    "operationSetCode": "NuGetVerify",
-                    "parameters": [ ],
-                    "toolName": "sign",
-                    "toolVersion": "1.0"
-                }
-            ]
-          SessionTimeout: 20
-
-      - task: PowerShell@2
-        displayName: "Get Hidi's version-number from .csproj"
-        inputs:
-          targetType: 'inline'
-          script: |
+resources:
+  repositories:
+  - repository: 1ESPipelineTemplates
+    type: git
+    name: 1ESPipelineTemplates/1ESPipelineTemplates
+    ref: refs/tags/release
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1ESPipelineTemplates
+  parameters:
+    pool:
+      name: Azure-Pipelines-1ESPT-ExDShared
+      image: windows-2022
+      os: windows
+    customBuildTags:
+    - ES365AIMigrationTooling
+    stages:
+    - stage: build
+      jobs:
+      - job: build
+        templateContext:
+          outputs:
+          - output: pipelineArtifact
+            displayName: 'Publish Artifact: Nugets'
+            artifactName: Nugets
+            targetPath: '$(Build.ArtifactStagingDirectory)/Nugets'
+          - output: pipelineArtifact
+            displayName: 'Publish Artifact: Hidi'
+            artifactName: Microsoft.OpenApi.Hidi-v$(hidiversion)
+            targetPath: '$(Build.ArtifactStagingDirectory)/Microsoft.OpenApi.Hidi-v$(hidiversion)'
+        steps:
+        - task: UseDotNet@2
+          displayName: 'Use .NET 6'
+          inputs:
+            version: 6.x
+
+        - task: UseDotNet@2
+          displayName: 'Use .NET 8'
+          inputs:
+            version: 8.x
+
+        - task: PoliCheck@2
```

Comment:
The tools like `PoliCheck`, `AntiMalware` and Binkim are automatically injected by the 1ES template and should be removed.

Reply:
Okay let me remove the corresponding tasks